lib/src/header_parser/clang_bindings/clang_bindings.dart

```diff
@@ -1706,6 +1837,24 @@ class CXChildVisitResult {
   static const int CXChildVisit_Recurse = 2;
 }
 
+class CXEvalResultKind {
```

Comment:
Maybe make it abstract it's clear that it only contains consts.

Reply:
I am not really sure what you mean.
Do you want to change how enums are generated by our tool?

Reply:
I meant `abstract class EnumName { ... }`.